build/fetch-cloudstate-pb.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```

Comment:
Should we include this in Travis' pipeline?

Reply:
same comment as fr compile protobuf script.